parquet-column/src/main/java/org/apache/parquet/io/MessageColumnIO.java

```diff
@@ -46,14 +46,13 @@
 import org.apache.parquet.io.api.RecordMaterializer;
 import org.apache.parquet.schema.MessageType;
 
+import it.unimi.dsi.fastutil.ints.IntArrayList;
 import static org.apache.parquet.Preconditions.checkNotNull;
 
 /**
  * Message level of the IO structure
  *
- *
```

Comment:
in general, try to not make whitespace changes in parts of the code that you didn't change, it makes the diff a little easier to read (just for future PRs)